scopes/react/react/webpack/webpack.config.base.ts

```diff
@@ -135,18 +135,6 @@ export default function (isEnvProduction = false): Configuration {
                 configFile: false,
                 customize: require.resolve('babel-preset-react-app/webpack-overrides'),
                 presets: [require.resolve('@babel/preset-react')],
-                plugins: [
-                  [
-                    require.resolve('babel-plugin-named-asset-import'),
-                    {
-                      loaderMap: {
-                        svg: {
-                          ReactComponent: '@svgr/webpack?-svgo,+titleProp,+ref![path]',
```

Comment:
Thanks @KutnerUri. I'm not familiar with the code base but it looks like you were using some svgr options in your previous config. So you may want to include these options in the svgr loader config. Example:

```js
{
  loader: '@svgr/webpack',
  options: {
    svgo: false,
    titleProp: true,
    ref: true,
  },
}
```

Reply:
ooh, thanks for the catch. I will add it

Reply:
I don't know why CRA has it disabled, might be related to https://github.com/facebook/create-react-app/pull/5062
If you want to enable it, I think you should have a look at its [default plugins](https://github.com/svg/svgo#built-in-plugins). For example `removeViewBox` plugin is enabled by default but some users might not want `viewBox` to be removed.

Reply:
ok. thanks :)